### PR TITLE
[Studio] CMS updates

### PR DESCRIPTION
**Pull request by wrpeck**
#### Description of changes:
Updating CMS documentation for the release of Data Manager

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [x] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [x] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [x] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br />
      _ref: MDX: `[link](https://link.com)`
            HTML: `<a href="https://link.com">link</a>`_
            
### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/console/data/content-management.mdx [hunk 1]
```diff
@@ -46,32 +45,60 @@ When you create a new instance in a table, you can also link it to an instance i
...
 3. On the **Actions** menu you have two options for downloading data.
-  1. Choose **Download selected items to CSV** to download only the selected rows of data.
-  2. Choose **Download results to CSV** to download all rows of data on the currently selected page.
+  1. Choose **Download selected items (.csv)** to download only the selected rows of data.
+  2. Choose **Download all items (.csv)** to download all rows of data on the currently selected model.
 4. Once you have selected a download option, your data should immediately start downloading as a CSV.
 
-## Create or edit data in markdown
-You can also edit and store the data as markdown with the markdown editor. This is especially useful for blogs, news, marketing, or content-focused apps where you want the app UI to be styled appropriately. The markdown editor is available by choosing **Edit in markdown** when you are in the **Edit** window for a data instance.
+## Limitations
```
Comment (ykethan) [inline src/pages/console/data/content-management.mdx:69]: Do we also want to add Seed data generation isn’t available if fields are required and of the following type: AWSPhone, Enum, Custom type, and Relationship.

File: src/pages/console/data/content-management.mdx [hunk 2]
```diff
@@ -46,32 +45,60 @@ When you create a new instance in a table, you can also link it to an instance i
...
 
-![cms](/images/console/cms.png)
+### Conflict Resolution
```
Comment (ykethan) [inline src/pages/console/data/content-management.mdx:71]: do we also want to add that IAM auth is needed for the data manager

File: src/pages/console/data/content-management.mdx (hunk 1 above)
Comment (ykethan) [reply src/pages/console/data/content-management.mdx:69]: additionally, I was thinking of adding the data store limitation. When we have multiple/duplicated tabs with studio the data manager will not load due to indexedDB. cc @abhi7cr for confirmation

Comment (wrpeck) [reply src/pages/console/data/content-management.mdx:69]: Added both of these callouts

**Review by ykethan — APPROVED** [review]
LGTM

**Review by bzsurbhi — APPROVED** [review]

**Review by jacoblogan — APPROVED** [review]

**Pull request merged**